build.gradle

```diff
@@ -98,15 +98,16 @@ tasks.withType(Sign).configureEach {
     notCompatibleWithConfigurationCache("$name task does not support configuration caching")
 }
 
-def gitHubReleaseName() {
-    return version.toString()
-}
```

Comment:
Either inline both gitHubReleaseName and gitReleaseTag, or none of them.

If you inline both, you can avoid the duplicate calls by getting the tagName in `createReleaseTag` from the `githubRelease` extension object. (it's surprising the git plugin does not wire the values like this by default)

Reply:
`gitHubReleaseName()` wasn't necessarily inlined - it's just not necessary any more because `releaseName` is being assigned to a `Provider<String>` instead of a `String`

Reply:
Done - the `tagName` in `createReleaseTag` now comes from the `githubRelease` extension object.